ApollosProject/apollos-prototype
# Removes unneeded FeedView padding

## DESCRIPTION

Android used to need some additional padding to make it happy. This doesn't appear to be needed anymore.

  
<img src="https://user-images.githubusercontent.com/2053547/64204696-4cad7b80-ce64-11e9-8579-20a6ad6b7bb3.png" width="50%" /><img src="https://user-images.githubusercontent.com/2053547/64204713-56cf7a00-ce64-11e9-9651-a41c9d29c93f.png" width="50%" />

### What design trade-offs/decisions were made?
N/A

### How do I test this PR?
- Poke around all the feeds in the app on android and ios and make sure everything looks 
- Look at the snapshot diffs and make sure only the necessary things changed.

### What automated tests did you write?
N/A

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
